degirum_tools/notifier.py

```diff
@@ -32,7 +34,13 @@ def __init__(
         *,
         message: str = "",
         holdoff: Union[Tuple[float, str], int, float] = 0.0,
-        token: Optional[str] = None,
+        # For now, token contains path to yaml. Eventually, it'll be the token and we can fetch the yaml from 
+        # the backend using this token as an identifier.
+        token: Optional[str] = None, 
+        # Tags needs to be in the proper "and" (using commas) or "or" (using spaces)
+        # ex: Tag1, Tag2 == Tag1 and Tag2

```

Comment:
Not clear, what this means. Should I pass the string "Tag1, Tag2 == Tag1 and Tag2" or you just trying to say that it can be either "Tag1, Tag2" or "Tag1 and Tag2" and these two ways of specifying tags are equivalent

Reply:
The latter. I can change the wording so that I say Tag1, Tag2 means Tag1 and Tag2.